src/Compilers/VisualBasic/Portable/VisualBasicExtensions.vb

```diff
@@ -266,6 +266,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return vbTree.GetPreprocessingSymbolInfo(identifierNode)
         End Function
 
+        <Extension>
+        Friend Function GetPreprocessingSymbolInfo(syntaxTree As SyntaxTree, identifierNode As SyntaxToken) As VisualBasicPreprocessingSymbolInfo
```

Comment:
identifierNode is a strange name for a token.  Also unclear if/why we need this given the helper above.